lib/config.json

```diff
@@ -0,0 +1,3 @@
+{
+    "credScanEnable": true
```

Comment:
you should put outside of lib folder, or is getting copied at build time?

Reply:
I've added it using `./config.json` so it's needed inside